projects/schematics/src/migrations/4_0/rename-symbol/rename-symbol.ts

```diff
@@ -0,0 +1,11 @@
+import { Rule, Tree } from '@angular-devkit/schematics';
+import { RenamedSymbol } from '../../../shared/utils/file-utils';
+import { migrateRenamedSymbols } from '../../mechanism/rename-symbol/rename-symbol';
+
+export const RENAMED_SYMBOLS_DATA: RenamedSymbol[] = [];
```

Comment:
It would be nice to have at least one referent example here. 
Is there a migration that you can do as part of this ticket?